frontend/src/components/website/adding_master/StepTrackingInfo.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+    <VContainer>
+        <VCard
+            flat
+            color="transparent"
+            class="form-card"
+        >
+            <VContainer pa-1>
+                <div class="subtitle-2 font-weight-bold">Tracking ID</div>
+                <VForm lazy-validation>
+                    <VCardText class="mb-2 pa-0 mt-2">
+                        <VTextField
+                            :value="trackingInfoId"
+                            single-line
+                            readonly
+                            solo
+                        />
+                    </VCardText>
+                    <VCardText class="mb-2 pa-0">
+                        <div class="subtitle-2 mb-2 font-weight-bold">Website tracking</div>
+                        <div class="caption mb-2">This is Global Site Tag tracking code for your website.</div>
+                        <VTextarea
+                            solo
+                            auto-grow
+                            readonly
+                            rows="6"
+                            value="<!--Global site tag - Analitycs-->
```

Comment:
move this to computed prop, Vue template should contains as less logic as possible